app/retail/emails.py

```diff
@@ -518,10 +518,39 @@ def render_funder_stale(github_username, days=60, time_as_str='a couple months')
     return response_html, response_txt
 
 
-def render_new_bounty(to_email, bounties, old_bounties, offset=3, trending_quests=[]):
+def render_new_bounty(to_email, bounties, old_bounties, offset=3, quest_of_the_day={}, upcoming_grant={}, upcoming_hackathon={}, latest_activities={}, from_date=date.today(), days_ago=7):
     from townsquare.utils import is_email_townsquare_enabled, is_there_an_action_available
-    email_style = (int(timezone.now().strftime("%-j")) + offset) % 24
+    from dashboard.models import Profile
+    from inbox.models import Notification
     sub = get_or_save_email_subscriber(to_email, 'internal')
+    if settings.DEBUG and False:
+        # for debugging email styles
+        email_style = 26
+    else:
+        email_style = (int(timezone.now().strftime("%-j")) + offset) % 24
+
+    # Get notifications count from the Profile.User of to_email
+    profile = Profile.objects.filter(email__iexact=to_email).last()
+    from_date = from_date + timedelta(days=1)
+    to_date = from_date - timedelta(days=days_ago)
+    notifications_count = Notification.objects.filter(to_user=profile.user.id, is_read=False, created_on__range=[to_date, from_date]).count()
```

Comment:
what if theres no profile? this will throw an exception

Reply:
But doesn't all emails that come at `to_email` has a profile?

Reply:
youd probably be better off passing the profile into the mail function instead of the email, or maybe both

Reply:
You mean to fetch the Profile id in `new_bounty_daily()` within `mails.py` and passing only the id into `render_new_bounty()`?

Reply:
Or maybe how about using just a `try..except: pas`s block while fetching the Profile and Notification?

Reply:
Will add a `try..except: pass` block.

Reply:
Done in [f37f851](https://github.com/gitcoinco/web/pull/6471/commits/f37f8512b0f0355974c30553b0f1bc3f32f3dc31)